src/NBitcoin/ConsensusOptions.cs

```diff
@@ -181,7 +181,7 @@ int maxStandardTxSigopsCost
         /// <param name="network">The network.</param>
         public virtual int GetStakeMinConfirmations(int height, Network network)
         {
-            if (network.Name.ToLowerInvariant().Contains("test")) // TODO: When rules are moved to network, we can use the extension method IsTest() from Stratis.Bitcoin.
```

Comment:
This is a consensus rule change please makes sure to properly test all scenarios (also gateway)